Utils/azuretools-core/src/com/microsoft/azure/toolkit/lib/common/task/AzureOperationEnhancer.java

```diff
@@ -20,45 +20,38 @@
  * SOFTWARE.
  */
 
-package com.microsoft.azure.toolkit.lib.common.operation;
+package com.microsoft.azure.toolkit.lib.common.task;
 
 import com.microsoft.azure.toolkit.lib.common.exception.AzureToolkitOperationException;
+import com.microsoft.azure.toolkit.lib.common.operation.AzureOperationRef;
 import lombok.extern.java.Log;
-import org.apache.commons.collections4.CollectionUtils;
 import org.aspectj.lang.JoinPoint;
 import org.aspectj.lang.annotation.*;
 import org.aspectj.lang.reflect.MethodSignature;
 
 import java.util.Objects;
-import java.util.logging.Level;
 
 @Aspect
 @Log
 public final class AzureOperationEnhancer {
 
-    @Pointcut("execution(@AzureOperation * *..*.*(..))")
+    @Pointcut("execution(@com.microsoft.azure.toolkit.lib.common.operation.AzureOperation * *..*.*(..))")
     public void operation() {
     }
```

Comment:
why for this change?

Reply:
seems it does not work sometimes if the name is not full.